tasks/main.yml

```diff
@@ -200,10 +200,12 @@
     name: glibc-common
     state: latest
   become: yes
+  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
```

Comment:
How about:
```suggestion
  when: not ansible_distribution == 'Debian'
```

Reply:
The problem here is, that a package with this name does not exist on debian (and i guess also ubuntu). Therefore i favor for white- instead of blacklisting.

Reply:
Fair enough :)